docs/contribute/index.md

```diff
@@ -10,20 +10,114 @@ Whether you're a technical writer or you've only edited Elastic docs once or twi
 
 ## Contribute to the docs [#contribute]
 
-The version of the docs you want to contribute to determines the tool and syntax you must use to update the docs.
+The version of the docs you want to contribute to determines the tool and syntax you must use to update the docs. The easiest way to find the source file for the page you want to update is to navigate to the page on the web and click **Edit this page**. This will take you to the source file in the correct repo. For more detailed instructions, see [Contribute on the web](on-the-web.md).
 
-### Contribute to Elastic Stack version 8.x docs and earlier
+### Contribute to version `8.x` docs and earlier
```

Comment:
unfortunately, this is not just stack 8.x and 9.x. we need to consider ECE 3.x vs. 4.x, ECK 2.x vs. 3.x ...

I believe there is a resource floating around somewhere that has the version cutoffs in it (Brandon would be able to help with that). We should link to that, and possibly provide a quick reference for common products here.	

`stack` is also an important word here that we need to preserve